### ISSUE TITLE
I keep getting error while trying to install node modules after cloning the repo

### ISSUE THREAD

**Issue by sabbCodes** [mozilla/perfcompare#307]
This is the error it brings;

npm install
[#########.........] - reify:@swc/core-android-arm-eabi: timing reifyNode:node_modules/@swc/core-android-arm64 Completed in 60[#########.........] - reify:@swc/core-android-arm-npm ERR! code ETIMEOUT
npm ERR! errno ETIMEOUT
npm ERR! request to https://registry.npmjs.org/yocto-queue/-/yocto-queue-0.1.0.tgz failed, reason: A "socket" was not created for HTTP request before 300001ms

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\devSabb\AppData\Local\npm-cache\_logs\2023-03-07T15_40_57_055Z-debug-0.log

I've tried checking my network and changed to different networks.
I also tried a different registry; npm config set registry https://registry.npmjs.org/
I'm not behind any proxy as well...
I also cleared the cache; npm cache clean --force

**Comment by Formasitchijoh** [comment]
which operating system are you using?

**Comment by sabbCodes** [comment]
![typescript](https://user-images.githubusercontent.com/109840351/223485452-d38b2b71-4424-4923-9fe1-a90148f0ee77.jpg)
![Outreachy typescript](https://user-images.githubusercontent.com/109840351/223485463-3e057cbe-fcea-4d74-8158-f8f7e9cc9594.jpg)


> which operating system are you using?

I use windows OS

**Comment by Formasitchijoh** [comment]
If you are using ubuntu make sure you have a stable internet connection to ensure a stable connection, also you should run as root so as to install all necessary dependencies

Use this command to disable any firewall : ufw disable

**Comment by sabbCodes** [comment]
Yes, it's the same network I used to clone the project. I also tried different networks

**Comment by Formasitchijoh** [comment]
Ok the @swc/core-android-arm-npm  is a typescript transpiler more of like Babel try install it using 

npm i @swc/core-android-arm-eabi

**Comment by sabbCodes** [comment]
I fixed it!!!!

It turned out that I was behind some proxy, I had to resolve it using; npm config delete proxy
Then tried again.
![Breakthrough Typescript](https://user-images.githubusercontent.com/109840351/223533047-fb5d985d-3140-40cb-90bc-6feba2dd4115.jpg)
